README.md

```diff
@@ -17,13 +17,13 @@ the square brackets (for multiple-choice questions).
 
 Select all that apply:
 ```
-[] Strings
-[] Booleans
-[] Undefined
+[x] Strings
+[x] Booleans
+[x] Undefined
 [] NaN
-[] Integers
+[x] Integers
```

Comment:
Almost! `Integer` actually isn't a data type -- `Number` is. Javascript is different from most other languages in that way.